chain/beacon/sync_manager.go

```diff
@@ -231,10 +256,19 @@ func (s *SyncManager) tryNode(global context.Context, upTo uint64, peer net.Peer
 				return false
 			}
 
-			if err := s.store.Put(beacon); err != nil {
-				logger.Debugw("unable to save", "with_peer", peer.Address(), "err", err)
-				return false
+			if force {
+				logger.Debugw("Using ForcePut to save beacon", "beacon", beacon.Round)
+				if err := s.store.ForcePut(beacon); err != nil {
+					logger.Debugw("unable to save", "with_peer", peer.Address(), "err", err)
```

Comment:
this seems like more than a debug
have some differentiation on the log message between the force put and the normal put case?